packages/moonstone/MoonstoneDecorator/MoonstoneDecorator.js

```diff
@@ -118,17 +120,113 @@ const MoonstoneDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	const Decorator = class extends React.Component {
 		static displayName = 'MoonstoneDecorator';
 
+		static contextTypes = contextTypes
+
+		static childContextTypes = contextTypes
+
+		static propTypes =  /** @lends moonstone/MoonstoneDecorator.AccessibilityDecorator.prototype */ {
+			/**
+			 * Enables additional features to help users visually differentiate components.
+			 * The UI library will be responsible for using this information to adjust
+			 * the components' contrast to this preset.
+			 *
+			 * @type {Boolean}
+			 * @public
+			 */
+			highContrast: PropTypes.bool,
```

Comment:
should `hightContrast` and `textSize` be a prop (runtime) of `MoonstoneDecorator` or a config (build time)?